charts/latest/csi-driver-smb/templates/csi-smb-node.yaml

```diff
@@ -103,7 +103,7 @@ spec:
           volumeMounts:
             - mountPath: /csi
               name: socket-dir
-            - mountPath: /var/lib/kubelet/
+            - mountPath: {{ .Values.kubelet.path }}
```

Comment:
don't need to change value here, it's inside container, let's keep `/var/lib/kubelet/`